x/stake/types/delegation.go

```diff
@@ -61,6 +61,7 @@ type UnbondingDelegation struct {
 	ValidatorAddr  sdk.Address `json:"validator_addr"`  // validator unbonding from owner addr
 	CreationHeight int64       `json:"creation_height"` // height which the unbonding took place
 	MinTime        int64       `json:"min_time"`        // unix time for unbonding completion
+	InitialBalance sdk.Coin    `json:"initial_balance"` // atoms initially scheduled to receive at completion
```

Comment:
Should this just be called the `Balance` instead of `InitialBalance`? really this term can get updated as a redelegation get's slashed - under that situation it would no longer hold the "initial balance".
Also the comment should be different
 - shouldn't reference atoms (just say tokens)
 - "scheduled to receive" -> "scheduled to finalize transfer"?

Reply:
We need to keep the `InitialBalance` around since the slashing amount is calculated according to what the balance was at the time of infraction (so if an `UnbondingDelegation` is slashed multiple times `Balance` will no longer be accurate).